console/frontend/src/main/frontend/src/app/views/status/status.component.html

```diff
@@ -41,6 +41,7 @@
       </li>
       <li
         *ngIf="messageLog[selectedConfiguration].errorStoreCount === -1"
+        data-cy-status-alert="true"
```

Comment:
Ik vind dit geen handig naam, en dit is volgens mij ook niet helemaal de juiste plek.
Er zijn heel veel warnings en errors die weergegeven kunnen worden.

Reply:
Opgelost in https://github.com/frankframework/frankframework/pull/6939/commits/02e3ae7a254823f0e268c955d80d21e47b43fa30
Het attribuut heet nu `data-cy-status-component="configuration-warnings"` en staat op de parent (`ul` element) van de configuration errors en warnings (`li`) elementen die direct onder de parent vallen)